packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -180,27 +314,69 @@ class DatePickerInput extends React.Component {
     const buttonText = intl.formatMessage({ id: 'Terra.datePicker.openCalendar' });
 
     return (
-      <div className={cx('custom-input')}>
-        <input
-          // Create a hidden input for storing the name and value attributes to use when submitting the form.
-          // The data stored in the value attribute will be the visible date in the date input but in ISO 8601 format.
-          data-terra-date-input-hidden
-          type="hidden"
-          name={name}
-          value={dateValue}
-        />
-        <Input
-          {...additionalInputProps}
-          className={inputClasses}
-          type="text"
-          name={'terra-date-'.concat(name)}
-          value={value}
-          onChange={this.handleOnChange}
-          placeholder={placeholder}
-          onFocus={onFocus}
-          onBlur={onBlur}
-          aria-label={intl.formatMessage({ id: 'Terra.datePicker.date' })}
-        />
+      <div
+        {...customProps}
+        className={dateInputContainerClasses}
+      >
+        <div className={dateInputClasses}>
+          <input
+            // Create a hidden input for storing the name and value attributes to use when submitting the form.
+            // The data stored in the value attribute will be the visible date in the date input but in ISO 8601 format.
+            data-terra-date-input-hidden
+            type="hidden"
+            name={name}
+            value={dateValue}
+          />
+          <Input
+            {...additionalInputProps}
+            className={cx('date-input-month')}
+            type="text"
+            name={'terra-date-month-'.concat(name)}
+            value={this.state.month}
+            onChange={this.handleMonthChange}
+            placeholder={dateMonth || 'MM'}
+            onFocus={this.handleOnFocus}
+            onBlur={this.handleOnBlur}
+            maxLength="2"
+            size="2"
+            pattern="\d*"
+            aria-label={intl.formatMessage({ id: 'Terra.datePicker.monthLabel' })}
+          />
+          <span className={cx('date-spacer')}>/</span>
```

Comment:
The date separator for all locales is a `/` with the exception of us-CA there the values are separated by `-`.

Reply:
Changed! [847d4f0](https://github.com/cerner/terra-framework/pull/1004/commits/847d4f05fecc8074996a2d504e3cc216973b6eef)